src/utils/loaders/google_map_loader.js

```diff
@@ -1,7 +1,8 @@
 import { isEmpty } from 'lodash';
-/* eslint-disable no-console */
+
 const BASE_URL = 'https://maps';
 const DEFAULT_URL = `${BASE_URL}.googleapis.com`;
+const API_PATH = '/maps/api/js?callback=_$_google_map_initialize_$_';
```

Comment:
Moving this string here, makes https://github.com/istarkov/google-map-react/pull/533/files#diff-4ed1519a6d20f85caa4cc6b843258ecbR86 way clearer.